src/api.ts

```diff
@@ -256,6 +256,25 @@ export interface QuoteRequest extends ToolConfiguration {
   insurance?: boolean // indicates whether the user wants a quote with bridge insurance
 }
 
+export interface QuoteToAmountRequest extends ToolConfiguration {
```

Comment:
Can you maybe do something like 

`QuoteToAmount = Omit<QuoteRequest, 'fromAmount'> & { toAmount: ... }`